README.md

```diff
@@ -11,56 +17,64 @@ Visit [Foxglove Docs](https://docs.foxglove.dev/) to get started.
 <tr><th>Package</th><th>Version</th><th>Description</th></tr>
 </thead>
 <tbody>
+
 <tr><td><strong>Python</strong></td><td></td><td></td></tr>
 <tr>
-<td>foxglove-schemas-flatbuffer</td>
 <td>
 
-[![pypi version](https://shields.io/pypi/v/foxglove-schemas-flatbuffer)](https://pypi.org/project/foxglove-schemas-flatbuffer/)
+[foxglove-sdk](https://pypi.org/project/foxglove-sdk/)
 
 </td>
-<td>Compiled flatbuffers</td>
-</tr>
-<tr>
-<td>foxglove-schemas-protobuf</td>
 <td>
 
-[![pypi version](https://shields.io/pypi/v/foxglove-schemas-protobuf)](https://pypi.org/project/foxglove-schemas-protobuf/)
+[![pypi version](https://shields.io/pypi/v/foxglove-sdk)](https://pypi.org/project/foxglove-sdk/)
 
 </td>
-<td>Compiled protocol buffers</td>
+<td>Foxglove SDK for Python</td>
 </tr>
+
+<tr><td><strong>C / C++</strong></td><td></td><td></td></tr>
 <tr>
-<td>foxglove-websocket</td>
 <td>
 
-[![pypi version](https://shields.io/pypi/v/foxglove-websocket)](https://pypi.org/project/foxglove-websocket/)
+[foxglove](./c)
 
 </td>
+<td></td>
+<td>Foxglove SDK for C</td>
+</tr>
+<tr>
 <td>
 
-Python implementation of the WebSocket protocol<br/>Repo: [foxglove/ws-protocol](https://github.com/foxglove/ws-protocol/tree/main/python)
+[foxglove_cpp](./cpp)
 
 </td>
+<td></td>
+<td>Foxglove SDK for C++</td>
```

Comment:
a bit early to add these...but whatever floats your boat

Reply:
yeah I was going to wait to merge this, but I might just remove them and merge it sooner

Reply:
ok I removed c/c++ and added an additional disclaimer about beta testing